Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:10-alpine3.10
+
+WORKDIR /opt/referee
+
+# Other dependencies
+RUN apk add python make bash
+
+COPY package.json yarn.lock lerna.json ./
+
+RUN yarn install
+RUN yarn bootstrap
+
+COPY . .
+
+RUN npm rebuild node-sass
```

Comment:
I think you should use `yarn` instead of `npm` here

Reply:
Apparently, there is an issue with yarn and node-sass, I can't say much about it since I'm not well versed on the node ecosystem, but apparently is a known issue, and the node-sass developers recommend using npm to rebuild it.
https://github.com/sass/node-sass/issues/1918
I am honestly impressed an issue from 2017 is still relevant here.